packages/firebase-frameworks/README.md

```diff
@@ -0,0 +1,135 @@
+# Firebase CLI & Web Frameworks
+
+## Frameworks
+
+| Framework  | Support | |
+| ---------- | ------- | - |
+| [Next.js](https://firebase.google.com/docs/hosting/frameworks/nextjs) | **Early preview** | |
+| [Angular](https://firebase.google.com/docs/hosting/frameworks/angular) | **Early preview** | |
+| [Express](https://firebase.google.com/docs/hosting/frameworks/express) | **Early preview** | |
+| Flask | **Early preview** | Coming soon... |
+| Django | Experimental | Coming soon... |
+| [Flutter](https://firebase.google.com/docs/hosting/frameworks/flutter) | Experimental | |
+| Nuxt | Experimental | |
+| Astro | Experimental | |
+| SvelteKit | Experimental | |
+| Preact<br>React<br>Lit<br>Svelte<br>and more... | Experimental | Static web apps, powered by *Vite* |
+
+## Overview
+
+Firebase Hosting integrates with popular modern web frameworks including Angular and Next.js. Using Firebase Hosting and
```

Comment:
Should this entire overview be retooled to refer to App Hosting instead of Firebase Hosting?

iiuc, we're going to keep "supporting" framework-aware CLI via the very similar docs on Devsite, while we basically transform this particular readme and repo to pertain to FAH adapters.  It will be key to distinguish the two arenas I think.

Reply:
This is just a copy and past of the old docs to the subsection that they apply to. I talked to James earlier and we decided we can just keep the old docs here for now too since they still make sense for this section. I don't think either of us felt very strongly about this though, so if you think it makes sense to remove this section as a whole I'm good with that too